smalltalksrc/VMMaker/SpurMemoryManager.class.st

```diff
@@ -12449,12 +12450,15 @@ SpurMemoryManager >> storePointer: fieldIndex ofObject: objOop withValue: valueP
 { #category : #'object access' }
 SpurMemoryManager >> storePointerImmutabilityCheck: fieldIndex ofObject: objOop withValue: valuePointer [
 	"Note must check here for stores of young objects into old ones."
-	<inline: true> "Must be inlined for the normal send in cannotAssign:to:withIndex:"
 
-	self cppIf: IMMUTABILITY ifTrue: 
-		[self deny: (self isImmediate: objOop).
-		 (self isImmutable: objOop) ifTrue: 
-			[^coInterpreter cannotAssign: valuePointer to: objOop withIndex: fieldIndex]].
+	"Must be inlined for the normal send in cannotAssign:to:withIndex:"
+
+	<inline: true>
+	self druidIgnore: [
```

Comment:
good! So this means that just the store + the write barrier for the remembered set take too much compilation place? interesting.